openfl-tutorials/experimental/workflow/404_Keras_MNIST_with_FedProx.ipynb

```diff
@@ -51,7 +51,7 @@
    "id": "4dbb89b6",
    "metadata": {},
    "source": [
-    "First we start by installing the necessary dependencies for the workflow interface. Note that this tuorial uses Keras 3, make sure you use python 3.9 or higher."
+    "First we start by installing the necessary dependencies for the workflow interface. Note that this tuorial uses Keras 3, make sure you use python 3.10 or higher."
```

Comment:
OpenFL is assumed to be installed at this point, which would not have happened without python requirements being met. Consider removing mention of python altogether.

Reply:
The steps mentioned below this comment helps user install Tensorflow, openfl and other required libs. If we are assuming openfl and required lib are installed then we should remove the steps mentioned below as well.

Reply:
First, pip verifies python version requirement for every package that is installed (which is why we mention in `setup.py`). That is an implicit check. Second, if required anyway, it is better to point the user to our official documentation - [Refer to the installation guide for OpenFL](https://openfl.readthedocs.io/en/latest/get_started/installation.html).

My broader point is to increase maintainability of the framework as part of the PR. 
It is one less line to maintain when we bump minimum python to 3.11 next year.